session-1-jigsaw-intro/09_Automodules/README.md

```diff
@@ -0,0 +1,96 @@
+### External libraries example
+
+In this example we include external libraries, which haven't been modularized so far.
+These jar files can be translated into so called *"automatic modules"* by `javac`.
```

Comment:
A link to some resource about automatic modules - so readers know what they are.